package.json

```diff
@@ -26,7 +26,7 @@
     "lint": "eslint --cache --ext .ts,.js,.mjs,.cjs . && prettier -c src test",
     "lint:fix": "eslint --cache --ext .ts,.js,.mjs,.cjs . --fix && prettier -c src test -w",
     "prepack": "pnpm run build",
-    "release": "pnpm test && changelogen --release && npm publish && git push --follow-tags",
+    "release": "pnpm test && changelogen --release && pnpm publish && git push --follow-tags",
```

Comment:
By the way, I also found how to call npm scripts differs from place to another: `pnpm <script>` in some places and `pnpm run <script>` in other places.

Is this intended or otherwise I suggest to unify the way (personally, `pnpm run <script>` is preferable because it's easy to distinguish custom npm scripts from pnpm commands).

Once you give me an answer, I will submit a PR.

Reply:
We can remove run for simplicity!